api/v1alpha1/clustergroupupgrade_types.go

```diff
@@ -70,8 +70,14 @@ type ClusterGroupUpgradeSpec struct {
 	// placement rules and placement bindings are created, but clusters are not added to the placement rule.
 	// Once set to true, the clusters start being upgrades, one batch at a time.
 	//+kubebuilder:default=true
-	Enable   bool     `json:"enable,omitempty"`
-	Clusters []string `json:"clusters,omitempty"`
+	Enable bool `json:"enable,omitempty"`
+	// This field determines whether container image pre-caching will be done on all the clusters
+	// matching the selector.
+	// If repuired, the pre-caching process starts immediately on all clusters irrespectively of
+	// the value of the "enable" flag and the remediation strategy
```

Comment:
The UOCR doesn't have a remediationStrategy anymore, so I think we can remove the last part of the comment.